pkg/utils/utils.go

```diff
@@ -311,3 +311,79 @@ func Remove(arr []string, item string) []string {
 	}
 	return arr
 }
+
+var GlobalK8NSObj *K8ValidNamespaces = &K8ValidNamespaces{}
```

Comment:
We are anyway using a getter for this variable. So, maybe we should make this variable private, if you don't want it to be manipulated directly?